src/pages/about.astro

```diff
@@ -25,254 +27,256 @@ const members = filterInactiveMembers(await getMembers());
+            Source software we all consume. Our goal is to establish a new social norm in the tech industry of companies
+            paying Open Source maintainers, so that burnout and related security issues such as those in XZ and Apache Log4j can
+            become a thing of the past.</p
+          >
+        </section>
+
+        <section id="what-is-the-pledges-mission">
+          <h2>What is the Pledge's mission?</h2>
+
+          <p
+            >A massive portion of the software we all depend on is based on Open Source projects. Though these projects
+            are very important, the people who tirelessly maintain them often do so
+            <a href="https://openpath.chadwhitacre.com/2024/the-open-source-sustainability-crisis/"
+              >without being paid at all</a
+            >. This means
+            that many maintainers have to either struggle to make ends meet, or work a second shift after working hours to
+            maintain projects that often have to be looked after for years or decades.</p
+          >
+
+            <figure class="text-center">
+              <img
+                src="/images/dependency.png"
+                alt="This xkcd comic shows a Jenga-like tower of blocks, illustrating “all modern digital infrastructure”. The structure precariously rests on a small load-bearing block, titled “a project some random person in Nebraska has been thanklessly maintaining since 2003”."
+                title="Someday ImageMagick will finally break for good and we'll have a long period of scrambling as we try to reassemble civilization from the rubble."
+              >
+              <figcaption><a href="https://xkcd.com/2347/">xkcd #2347 — Dependency</a></figcaption>
+            </figure>
+
+          <p
+            >This arrangement isn't fair to the maintainers who build software we value. And it leads to a fragile
+            ecosystem that's susceptible to security issues, because the people who look after our digital infrastructure
+            are likely to suffer from burnout. This has led to issues such as the
+            <a href="https://en.wikipedia.org/wiki/XZ_Utils_backdoor">XZ backdoor</a>, the
+            <a href="https://en.wikipedia.org/wiki/Log4Shell">Log4Shell vulnerability</a>, and many other serious
+            issues.</p
+          >
+
+          <p
+            >This is why our main goal is <strong>to get companies to pay maintainers</strong> of software they depend on.
+            We believe that by doing this, we can contribute to a healthy, thriving and secure Open Source ecosystem. We
+            recognise this is a multifaceted problem, but the solution we are focusing on is providing <strong>direct
+            payments to maintainers</strong>, in order to support them and their work.</p
+          >
+        </section>
+
+        <section>
+          <h2>What does joining the Pledge entail?</h2>
```

Comment:
Is there a reason not all FAQs are included in the TOC?

Reply:
Ah shoot I think i messed up when resolving merge conflicts. will push a fix for this tomorrow!

Reply:
Pushed a commit for this!